module/zfs/vdev_draid.c

```diff
@@ -2200,8 +2229,13 @@ vdev_draid_fini(vdev_t *vd)
 {
 	vdev_draid_config_t *vdc = vd->vdev_tsd;
 
+#ifdef	__dilos__
+	kmem_free(vdc->vdc_perms, sizeof (uint8_t) *
+	    vdc->vdc_children * vdc->vdc_nperms);
+#else
 	vmem_free(vdc->vdc_perms, sizeof (uint8_t) *
 	    vdc->vdc_children * vdc->vdc_nperms);
+#endif
```

Comment:
Let's call the new helper vdev_draid_free_perms() function here since it was added for this purpose.